packages/tbds-support/lib/layout.scss

```diff
@@ -0,0 +1,14 @@
+$tbds-space-x-small: 0.25rem !default;
+$tbds-space-small: 0.5rem !default;
+$tbds-space-medium: 1rem !default;
+$tbds-space-large: 1.5rem !default;
+$tbds-space-x-large: 2rem !default;
+$tbds-space-xx-large: 3rem !default;
+
+// OR, more flexible with ability to expand/contract…
+$tbds-space-0.25x: 0.25rem !default;
+$tbds-space-0.5x: 0.5rem !default;
+$tbds-space-1x: 1rem !default;
+$tbds-space-1.5x: 1.5rem !default;
+$tbds-space-2x: 2rem !default;
+$tbds-space-3x: 3rem !default;
```

Comment:
I love that the `x`'s as suffix allow for machine alphabetization (although it does not read right in English, but it does in Spanish and other romance languages). 

People might hate me, but I would love something like: `$tbds-space-large-xx`. The modifier is second, allowing for for alphabetization. Primary English speakers might have a moment of cognitive friction, but I think it's worth it, and it's easy to get used to.

Reply:
Or even `--xx`

Reply:
It also allows for "insertion" of new variants based off the "steps" of `large`, `small` etc. Although they may cross in value, it may not be an absolute negative.